kafka-agent/src/main/java/io/strimzi/kafka/agent/KafkaAgentUtils.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.agent;
+
+import io.fabric8.kubernetes.api.model.Secret;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.security.KeyFactory;
+import java.security.KeyStore;
+import java.security.PrivateKey;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.util.Base64;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+
+/**
+ * Class with various utility methods for generating KeyStore and TrustStore for KafkaAgent
+ */
+public class KafkaAgentUtils {
```

Comment:
A lot of these methods are duplicates of ones we have elsewhere, but I assume you put them here so we didn't have to pull in any other Strimzi modules and create cyclic dependencies?